src/gatling/simulations/scenarios/NFD_DivorceApplication.scala

```diff
@@ -93,42 +93,42 @@ object NFD_DivorceApplication {
         .headers(CommonHeader)
         .headers(PostHeader)
         .formParam("_csrf", "${csrf}")
-        .formParam("inTheUk", Case.YesOrNo.No)
-        .check(substring("Is your original marriage certificate in English?")))
+        .formParam("inTheUk", Case.YesOrNo.Yes)
+        .check(substring("Where you got married")))
     }
     .pause(MinThinkTime seconds, MaxThinkTime seconds)
 
-    .group("DivorceApp_080_OriginalMarriageCertificateSubmit") {
-      exec(http("Original Marriage Certificate Submit")
-        .post(BaseURL + "/certificate-in-english")
+    .group("DivorceApp_080_WhereYouGotMarriedSubmit") {
+      exec(http("Where You Got Married Submit")
+        .post(BaseURL + "/country-and-place")
         .headers(CommonHeader)
         .headers(PostHeader)
         .formParam("_csrf", "${csrf}")
-        .formParam("certificateInEnglish", Case.YesOrNo.No)
-        .check(substring("Do you have a ‘certified translation’ of your marriage certificate?")))
+        .formParam("ceremonyCountry", "France")
```

Comment:
Surely if inTheUk = Yes on the previous request, it wouldn't ask you for the country on the following page, would it? Is this step still relevant, or should it be removed? I'm not sure of the flow.